ceph.spec.in

```diff
@@ -1463,6 +1464,8 @@ cmake .. \
 %if 0%{with seastar}
     -DWITH_SEASTAR:BOOL=ON \
     -DWITH_JAEGER:BOOL=OFF \
+    -DCMAKE_C_COMPILER=clang \
+    -DCMAKE_CXX_COMPILER=clang++ \
```

Comment:
seems we also need to add `%global toolchain clang`. i am adding it like:
```diff
diff --git a/ceph.spec.in b/ceph.spec.in
index 281db563bc0..262f03c6e28 100644
--- a/ceph.spec.in
+++ b/ceph.spec.in
@@ -1335,6 +1335,9 @@ This package provides a Ceph hardware monitoring agent.
 %if ( 0%{?rhel} && 0%{?rhel} < 9 ) || ( 0%{?suse_version} && 0%{?suse_version} <= 1500 )
 %define _lto_cflags %{nil}
 %endif
+%if 0%{with seastar}
+%global toolchain clang
+$endif

 %if 0%{with cephfs_java}
 # Find jni.h
```
see https://src.fedoraproject.org/rpms/redhat-rpm-config/blob/4637e1bd5512b869bd07c58b7545d2528a9bc4c8/f/macros#_44